contents/blog/is-google-analytics-illegal-microsite.md

```diff
@@ -0,0 +1,142 @@
+
+## A timeline
+
+### It all started when our CTO shared the news on Slack
+
+![The Netherlends also likely to declare Google Analytics illegal](../images/blog/is-ga-illegal-microsite/illegal-in-austria.png)
+
+### Then our VP of Ops came up with an idea to launch a microsite
+
+![Is this idea funny or just really dumb?](../images/blog/is-ga-illegal-microsite/is-this-funny-or-just-really-dumb.png)
+
+### Obviously this felt like a fun idea, so I put together a quick joke of a wireframe
+
+![Is GA illegal?](../images/blog/is-ga-illegal-microsite/is-it-illegal-probably.png)
+
+### Soon we had a gimmicky domain name and the idea was born
+
+![Greatest domain name ever](../images/blog/is-ga-illegal-microsite/is-google-analytics-illegal.png)
+
+### I remember being on a flight at the time. What better way to spend a couple hours in coach than to whip up a mock?
+
+![1 hour mockup](../images/blog/is-ga-illegal-microsite/mockup.png)
+
+## How we positioned the site
+
+We planned to launch this site on Hacker News, a site known for having an audience that really doesn’t like being sold to. If it doesn’t come across as authentic, be prepared to be torn to shreds.
+
+Thankfully one of our marketing values is “No sneaky shit.” I can’t say it better than it’s already stated [in our Company Handbook](https://posthog.com/handbook/growth/marketing#no-sneaky-shit):
+
+> Our focus customers are technical and acutely aware of the tedious clickbaity marketing tactics that software companies use to try and entice them. Stop. It's patronizing to both the customer and the marketing people creating the content.<br /><br />
+> We will never try to trick someone into using PostHog through spammy marketing tactics. That's a great way to boost vanity signup metrics in the short term before the customer quickly churns out.
+
+### Our challenges
+
+How can we:
+
+- have a little fun with this site concept
+- provide real value to our visitors
+- see an uptick in traffic to PostHog.com
+- make it obvious we’re not trying to make this solely a marketing stunt?
+
+### 1. Provide value: **Include our competitors and let anyone add to the list**
+
+We seeded our [Google Analytics alternatives](https://isgoogleanalyticsillegal.com/alternatives) page with all of the competitors we could think of who would solve this problem for companies affected by this ruling:
+
+- Open source analytics tools
+- Self-hostable analytics tools
+- Analytics providers in the EU, where customer data isn’t transmitted to the United States
+
+We also [published our codebase on GitHub](https://github.com/PostHog/isgoogleanalyticsillegal.com), where anyone can submit a pull request to add another competitor to the list.
+
+### 2. Don’t take it too seriously
+
+When you hover over the map of countries where Google Analytics has been deemed illegal, you’ll see a photo of Ursula von der Leyen, the President of the European Commission, inferring her enthusiasm about the rulings.
```

Comment:
'indicating her enthusiasm' would be better (inferring doesn't quite make sense here)